corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -214,6 +215,21 @@ def test_include_without_expanding(self, uses_locations):
             ['Massachusetts', 'Suffolk', 'Boston', 'Revere', 'New York']
         )
 
+    def test_include_without_expanding_same_level(self, uses_locations):
+        # I want a list of all the counties, but only the cities in my county
+        self.user._couch_user.set_location(self.locations['Boston'].couch_location)
+        location_type = self.locations['Boston'].location_type
+
+        # Get all the counties
+        location_type.include_without_expanding = self.locations['Middlesex'].location_type
```

Comment:
You can also do
```python
location_type = self.location_types['county']
```
(this is fine too, just wasn't sure if you knew that was available)